app/mailers/administrator_notifications/channel_notifications_mailer.rb

```diff
@@ -15,6 +15,14 @@ def facebook_disconnect(inbox)
     send_mail_with_liquid(to: admin_emails, subject: subject) and return
   end
 
+  def email_disconnect(inbox)
+    return unless smtp_config_set_or_development?
+
+    subject = 'Your Email Inbox configuration is not working'
```

Comment:
Your email inbox has been disconnected. Please update the credentials for SMTP/IMAP.